tracer/src/Datadog.Tracer.Native/debugger_method_rewriter.cpp

```diff
@@ -1997,6 +2071,14 @@ HRESULT DebuggerMethodRewriter::Rewrite(RejitHandlerModule* moduleHandler,
     std::vector<TypeSignature> methodArguments = caller->method_signature.GetMethodArguments();
     int numArgs = caller->method_signature.NumberOfArguments();
 
+    if (caller->type.name.rfind(L'@') != std::wstring::npos)
```

Comment:
just for curiosity, `type.name` is a `std::wstring`, shouldn't it be a `WSTRING` instead ?
(it's probably an already existing, but just checking)

Reply:
`WSTRING::npos` and `std::wstring::npos` are both interchangeable - they are both boil down to `xstring`.
But I agree with your point of sticking with the same type. I will change it to `WSTRING::npos` 👍

Reply:
oh yeah,no need to change, it was mainly out of curiosity.